config/locales/en.yml

```diff
@@ -192,6 +192,20 @@ en:
 
       <h2 class="govuk-heading-m">If you didn’t expect to see this page</h2>
       <p class="govuk-body">You may be seeing this page because you’ve turned off cookies in your browser. You’ll need to <a class="govuk-link" href="https://ico.org.uk/your-data-matters/online/cookies/" rel="external" target="_blank">turn cookies on</a> before you can use this service.</p>
+  sms:
+    confirmation:
+      body_text: |
+        Hello %{first_name} %{last_name} - we’ve received your registration.
+
+        You don’t need to do anything else if you’ve had a letter from the NHS or your doctor confirming that you’re classed as clinically extremely vulnerable to coronavirus.
+
+        If you haven’t had a letter, contact your GP as soon as possible. Otherwise you may not get support.
+
+        We’ll check that you’re eligible, then you’ll start getting support if you asked for it. This may take time.
+
+        If you need urgent support and can’t rely on friends or neighbours, contact your local authority.
+
+        If your support needs change, go to https://www.gov.uk/coronavirus-extremely-vulnerable and answer the questions again.
```

Comment:
Do you know if this'll be one text based on the length? Just had a thought we might want a test for the length of this content, just in case.

Reply:
It will cost multiple text messages to send (5x SMSs IIRC), but it will be delivered as a single message to the recipient's device

Reply:
Wow big. That's good to know 👍